src/AdvancedHMC.jl

```diff
@@ -30,7 +30,16 @@ include("utilities.jl")
 # r: momentum variables
 # z: phase point / a pair of θ and r
 
-include("kinetic.jl")
+# TODO Move it back to hamiltonian.jl after the rand interface is updated
+abstract type AbstractKinetic end
+
+struct GaussianKinetic <: AbstractKinetic end
+
+export GaussianKinetic
+
+include("experimental/relativistic_hmc.jl")
```

Comment:
How about only loading experimental algorithms manually to avoid any additional dependency? For tests on CI, we can explicitly call `include("experimental/relativistic_hmc.jl")` from the test script. 
```suggestion
```